libs/client-api/src/v2/actor.rs

```diff
@@ -76,6 +81,20 @@ impl WorkspaceControllerActor {
     actor
   }
 
+  /// Return current changed collab ids and clear the set.
+  pub async fn consume_latest_changed_collabs(&self) -> HashSet<ChangedCollab> {
```

Comment:
1. This method doesn't have to be asynchronous.
2. Why use map? If you want to track the changed collabs, you could use `tokio::sync::broadcast::channel` and just consume it with await using `recv` or `try_recv`.

Reply:
flutter client periodically batch consume changed collabs. Any benefit by using channel for this case?